src/VisualStudio/Setup.Next/codeAnalysisService64.servicehub.service.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "host": "desktopClr",
+  "hostId": "RoslynCodeAnalysisService64",
```

Comment:
"desktopClr" service hub host is AnyCPU, not strictly x64. So naming the service "RoslynCodeAnalysisService64" may be confusing, especially when it runs on x86 machines.

Reply:
I will remove 64.